hironx_moveit_config/test/test_hironx_moveit.py

```diff
@@ -58,7 +58,6 @@ def __init__(self, *args, **kwargs):
     @classmethod
     def setUpClass(self):
 
-        rospy.init_node("test_hironx_moveit")
```

Comment:
I'm not sure why removing this line solves the exception "`rospy.init_node() has already been called with different arguments`".